### ISSUE TITLE
"Proton 6.21-GE-1 is not he current proton version in your prefix"

### ISSUE THREAD

**Issue by blake0812** [Moosatronic/LinuxFFXIVProtonFix#6]
I am getting this error when running the script, I have selected Proton-6.21-GE-2 but it is asking for GE-1. Also 
"grep: /home/name/.steam/steam/compatdata/3210/version: no such file or directory" this directory does in fact exist.

**Comment by Centzilius** [comment]
The file `/home/name/.steam/steam/compatdata/39210/version` must indeed match `6.21-GE-1` although it is `6.21-GE-2` (see the `CURRENT_PREFIX_VERSION` env var in the proton file from the archive). Seems to be a bug in the Proton version that does not really make a difference. Unless you made a copy paste error the app id in that path is incorrect although I wouldn't know how that happened. Out of interest. Where do you have your ffxiv installed?

**Comment by Kelseer** [comment]
Yeah I'm getting this error as well, but also at the Proton 6.3-3/8 step.  Using the steam deck.

This is my experience:
It says to please create a new prefix by forcing compatibility to proton 6.3.8.  I did so, started FFXIV and it showed the unusable log in screen.  I closed the game and pressed enter on the script and it says "Proton 6.3-3" is not the current proton version in your prefix and to press enter to leave the script but it continues on anyway.

Says to move to Proton 6.21-GE-2.  I did.  Same error as above.  

My guess is that the prefix it's looking for is not getting generated or is not being generated in the expected location.  In this case it seems to be looking for FFXIV in the compatdata folder but Steam, in it's infinite wisdom, decided to install it in /home/deck/.local/share/steam/steamapps/common.  There is a directory in compatdata but there is nothing about a versions folder.  I wonder if it's something about how SteamOS deals with it vs other installs?

**Comment by Moosatronic** [comment]
Could be the script made a mistake. I would check the file called version
in your prefix to see if it refers to 6.21 GE. The script is pretty
falliable as it just checks if certain folders or documents exist.

**Comment by ldv8434** [comment]
The grep error is caused by the incorrect directory mentioned in #4 . However, the user also needs to launch the game in steam a second time after changing their proton prefix to 6.21-GE-2 so that the version file will update.
